app/controllers/groups_controller.rb

```diff
@@ -1,13 +1,10 @@
 class GroupsController < ApplicationController
   def index
-  	@groups = Group.all
+    @groups = Group.all
   end
 
   def show
-  	# @group = Group.first
```

Comment:
I think Nikita created this file, it doesn't exist in my Ashby-village folder. I have no idea how I pushed it up?